Header header
bool command_flag  # 0 = pivot stationary, 1 = pivot slide